libs/crypto/keys_manager_impl.cpp

```diff
@@ -52,66 +52,51 @@ namespace iroha {
     auto sig = iroha::sign(
         iroha::sha3_256(test).to_string(), keypair.pubkey, keypair.privkey);
     if (not iroha::verify(
-        iroha::sha3_256(test).to_string(), keypair.pubkey, sig)) {
+            iroha::sha3_256(test).to_string(), keypair.pubkey, sig)) {
       log_->error("key validation failed");
       return false;
     }
     return true;
   }
 
-  nonstd::optional<iroha::keypair_t> KeysManagerImpl::loadKeys() {
-    // Try to load from local file
-    auto loadFile = [this](auto filename, auto &res) {
-      return [that = this, &filename, &res](auto keypair)
-          ->nonstd::optional<iroha::keypair_t> {
-        std::ifstream file(filename);
-        if (not file) {
-          that->log_->error("Cannot read '" + filename + "'");
-          return nonstd::nullopt;
-        }
-        file >> res;
-        return keypair;
-      };
-    };
+  bool KeysManagerImpl::loadFile(const std::string &filename,
+                                 std::string &res) {
+    std::ifstream file(filename);
+    if (not file) {
+      log_->error("Cannot read '" + filename + "'");
+      return false;
+    }
+    file >> res;
+    return true;
+  }
 
-    std::string client_pub_key;
-    std::string client_priv_key;
+  nonstd::optional<iroha::keypair_t> KeysManagerImpl::loadKeys() {
+    std::string pub_key;
+    std::string priv_key;
 
-    //TODO 15/11/17 motxx - better way to suppress the evaluation?
-    auto blocking = nonstd::make_optional<iroha::keypair_t>()
-        | loadFile(account_name_ + ".pub", client_pub_key)
-        | loadFile(account_name_ + ".priv", client_priv_key);
+    if (!loadFile(account_name_ + ".pub", pub_key)
+        || !loadFile(account_name_ + ".priv", priv_key))
```

Comment:
Style issue: we use alternative operators for boolean operations. Such as `or` and `not`.